sycl/test-e2e/AddressSanitizer/misaligned/misalign-long.cpp

```diff
@@ -0,0 +1,35 @@
+// REQUIRES: linux, cpu
+// RUN: %{build} %device_asan_flags -O0 -g -o %t
+// RUN: env SYCL_PREFER_UR=1 %{run} not %t 2>&1 | FileCheck %s
+// RUN: %{build} %device_asan_flags -O1 -g -o %t
+// RUN: env SYCL_PREFER_UR=1 %{run} not %t 2>&1 | FileCheck %s
+// RUN: %{build} %device_asan_flags -O2 -g -o %t
+// RUN: env SYCL_PREFER_UR=1 %{run} not %t 2>&1 | FileCheck %s
+#include <sycl/detail/core.hpp>
+#include <sycl/usm.hpp>
+
+#include <random>
+
+int main() {
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_int_distribution<> distrib(1, 7);
```

Comment:
Why do we need to randomize this test? Couldn't we just test for different offset?

Reply:
I think it's fine to use random here.